### DISCUSSION TITLE
v2.12.7 breaking change (in tests)

### DISCUSSION THREAD

**Discussion by jonneroelofs** [livewire/livewire#8643] [7. Bugs (archived)] [answered]
### Livewire version

v2.12.7

### Laravel version

v9.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

Could not tell you how to reproduce it unfortunately but when I run a test suite with 295 tests a specific one of them breaks with a  Undefined array key "fingerprint" after upgrading from v2.12.6 to v2.12.7. The issue doesn't show itself if you run the failing tests separately so that makes it really hard to debug.

Reverting to v2.12.6 resolves the issue.
My money is on this PR: https://github.com/livewire/livewire/pull/6512

![image](https://github.com/livewire/livewire/assets/20800818/529d7e43-d5bc-4e8c-b2ca-e030193e2546)

### Repository or Wirebox URL

https://apologies-have-no-url.sorry

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@jonneroelofs thanks for reporting. I've messaged Caleb to see what he wants to do about it, because that PR was merged in September, but evidently never got tagged. So it might just be worth reverting that PR.

But if you can manage to replicate the issue, that would help us look into it further to see if we can fix it without reverting.

Thanks!

**Reply by joshhanley** [reply, marked as answer]
@jonneroelofs it has been reverted and a new release v2.12.8 has been tagged 🙂

**Reply by jonneroelofs** [reply]
you are the best
